src/services/Messaging.cpp

```diff
@@ -339,4 +339,37 @@ std::string Messaging::createSubscribers(const std::string &topicId,
                                 std::to_string(statusCode) +
                                 "\n\nResponse: " + response);
     }
-}
\ No newline at end of file
+}
+std::string Messaging::updateMessage(const std::string &messageId,
+                                     const std::string &subject,
+                                     const std::string &content) {
+    if (messageId.empty()) {
+        throw AppwriteException("Missing required parameter: 'messageId'");
+    }
+    if (subject.empty()) {
+        throw AppwriteException("Missing required parameter: 'subject'");
+    }
+    if (content.empty()) {
+        throw AppwriteException("Missing required parameter: 'content'");
+    }
+
+    std::string url = Config::API_BASE_URL + "/messaging/messages/" + Utils::urlEncode(messageId);
+
+    std::string payload = R"({"subject":")" + Utils::escapeJsonString(subject) +
+                          R"(","content":")" + Utils::escapeJsonString(content) + R"("})";
+
+    std::vector<std::string> headers = Config::getHeaders(projectId);
```

Comment:
To use api key and project id here, you need to initiate appwrite object in your example file.